tox.ini

```diff
@@ -109,7 +109,7 @@ deps =
     Twisted[http2]==18.9.0
     w3lib==1.17.0
     zope.interface==5.1.0
-    lxml==4.4.1
+    lxml==4.6.0
```

Comment:
This should be synced with version restrictions in setup.py.

Reply:
Done.